packages/core/resolve-client/test/unit/client.unit.test.ts

```diff
@@ -182,6 +182,33 @@ describe('command', () => {
       }
     )
   })
+
+  test('bug fix: #1629', (done) => {
+    mRequest.mockRejectedValueOnce(Error('error'))
+
+    let callbackError: any = null
+
+    client.command(
+      {
+        aggregateName: 'user',
+        aggregateId: 'user-id',
+        type: 'create',
+        payload: {
+          name: 'user-name',
+        },
+      },
+      (error) => {
+        if (error) {
+          callbackError = error
+        }
+      }
+    )
+
+    setTimeout(() => {
+      expect(callbackError).toEqual(Error('error'))
+      done()
+    }, 10)
```

Comment:
10 milliseconds is magic constant. If just next tick domain callback is needed, function like `setImmediate` or `Promise.resolve().then(Promise.resolve).then()` will be better. If real delay is needed construction like this is unstable